ganga/GangaCore/Lib/Localhost/Localhost.py

```diff
@@ -183,11 +186,21 @@ def preparejob(self, jobconfig, master_input_sandbox):
         from GangaCore.Core.Sandbox.WNSandbox import PYTHON_DIR
         import inspect
 
-        fileutils = File( inspect.getsourcefile(GangaCore.Utility.files), subdir=PYTHON_DIR )
+        virtualization = job.virtualization
 
+        utilFiles= []
+        fileutils = File( inspect.getsourcefile(GangaCore.Utility.files), subdir=PYTHON_DIR )
+        utilFiles.append(fileutils)
+        if virtualization:
+            virtualizationutils = File( inspect.getsourcefile(GangaCore.Utility.Virtualization), subdir=PYTHON_DIR )
+            utilFiles.append(virtualizationutils)
+            if type(virtualization.image) is File:
+                utilFiles.append(virtualization.image)
+            if type(virtualization.image) is LocalFile:
+                utilFiles.append(File(virtualization.image.location()[0]))
```

Comment:
This will only work for `LocalFile` and not for any of the other `GangaFile` style objects. We should use the general machinery as done for `j.inputfiles`.

Reply:
At the moment left it as the user responsibility to add file to the inputfiles.